src/pages/policies-and-audits/audits/[...slug].astro

```diff
@@ -0,0 +1,34 @@
+---
+import { getCollection } from 'astro:content';
+import PageLayout from '../../../layouts/PageLayout.astro';
+import SideNav from '../../../components/SinglePageSideNav.astro'
+import TopNav from '../../../components/TopNav.astro'
+export async function getStaticPaths() {
+  const entries = await getCollection('audits');
+  return entries.map(entry => {
+    const slug = entry.slug == './' ? undefined : entry.slug
+    return {
+      params: { slug: slug},
+      props: { entry, entries}
+    }
+  })
+}
+const { entry, entries} = Astro.props;
```

Comment:
I don't think we use `entries` in the component. Can we remove it?

Reply:
sure, will remove it